src/content/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/understand-technical-concepts/streaming-alerts-key-terms-concepts.mdx

```diff
@@ -111,6 +111,17 @@ All of these configurable delays give you more control over how you're alerted o
       </td>
     </tr>
 
+    <tr>
+      <td>
+        Sliding windows
+      </td>
+      <td>
+        When enabled, it causes aggregation windows to overlap, creating smoother charts.
+
+        The sliding windows duration controls the amount of time your aggregation windows overlap.
```

Comment:
"determines" instead of "controls"?

Reply:
I rewrote this sentence to clarify what's going on.